CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,66 @@
+
+## Our expectations
+
+To make clear what is expected, we ask all members of the community to conform to the following Code of Conduct.
+
+* All communication - online and in person - should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate at any time.
+
+* Be kind to others. Do not insult or put down other contributors.
+
+* Behave professionally. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate.
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Harassment includes offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of discussions, inappropriate physical contact, and unwelcome sexual attention.
+
+Examples of unacceptable behavior by community members include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional setting
+
+***Participants asked to stop any harassing behavior are expected to comply immediately.***
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+
+## Enforcement
+
+Members of the community who violate these rules - no matter how much they have contributed to the BIDS Starter Kit, or how specialised their skill set - will be approached by Kirstie Whitaker. If inappropriate behaviour persists after a discussion with Kirstie, the contributer will be asked to discontinue their participation in the BIDS Starter Kit project.
+
+**To report an issue** please contact [Kirstie Whitaker][kirstie-github]. All communication will be treated as confidential.
```

Comment:
Although this might be a bit unlikely, for new contributors I'd just like to be explicit that this is *an issue related to the code of conduct* or a *potential code of conduct violation* rather than simply an "issue", since I could see that getting confused with GitHub issues ! WDYT ?

Reply:
Yep - that seems sensible.

Does "To report a problem" work instead?

Reply:
Yes, perfect ! Thank you